config/config.exs

```diff
@@ -1,3 +1,25 @@
 import Config
 
 config :phoenix, :json_library, Jason
+alias LazyDoc.Providers.GithubAi
```

Comment:
if possible I would like to not use any configuration. Maybe this library can offer all those defaults internally? So I don't need to add any config at the start.
As for the env var, this can still be done internally in the lib, by using `System.get_env("GITHUB_API_TOKEN")` for example (and `Application.get_env` as fallback).
This would make the config less bloated, I would like to minimize any config on this project and the only reason  I added is because Phoenix require the json_library to be mapped

Reply:
Yup, `lazy_doc` offers defaults to everything except for the token, so most of the configuration can be removed.